### PR TITLE
docs: Update book to sylvia 1.1.0

### LINKED ISSUE #12: Update sylvia-book
Add:
 * `sudo` entry-point
 * auto-generated `trait Executor`
 * new attributes `sv::attr` and `sv::msg_attr`

### PR DESCRIPTION
**Pull request by kulikthebird**

### CODE REVIEW

File: src/advanced/attributes_forwarding.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Attributes forwarding
+
+This feature allows ^sylvia users to forward any attribute to every message
```
Comment (jawoznia) [inline src/advanced/attributes_forwarding.md]:
```suggestion
This feature allows ^sylvia users to forward any attribute to specific message
```

File: src/advanced/attributes_forwarding.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+type using `#[sv::msg_attr(msg_type, ...)]` attribute. For the messages
+that resolves to enum types it is possible to forward attributes per
+enum field by using `#[sv::attr(...)]` - this works for `exec`, `query`
+and `sudo` methods.
```
Comment (jawoznia) [inline src/advanced/attributes_forwarding.md]:
```suggestion
type using `#[sv::msg_attr(msg_type, ...)]` attribute.
For the messages that resolves to enum types it is possible to forward attributes to their specific variants by using `#[sv::attr(...)]` on top of the appropriate method - this works for `exec`, `query`
and `sudo` methods.
```

File: src/advanced/custom.md [hunk 3]
```diff
@@ -476,14 +439,15 @@ Running `poke` on our contract will send the `ExternalMsg::Poke`, which `App` wi
...
 use sylvia::multitest::App;
 
-use crate::contract::sv::mt::{CodeId, CounterContractProxy};
+use crate::contract::sv::mt::{CodeId, CustomContractProxy};
 use crate::multitest::custom_module::CustomModule;
 
 #[test]
 fn test_custom() {
-    let owner = "owner".into_addr();
+    let owner = "owner".into_bech32();
```
Comment (jawoznia) [inline src/advanced/custom.md:450]: both should be ok here, but we can change it.

File: src/advanced/entry_points_overriding.md [hunk 4]
```diff
@@ -1,29 +1,30 @@
...
 ## Example
 
-To make ^sylvia generate multitest helpers with `sudo` support, you first need to define your
+To make ^sylvia generate multitest helpers with `cusom_entrypoint` support, you first need to define your
 `entry point`.
 
 ```rust,noplayground
 #[entry_point]
-pub fn sudo(deps: DepsMut, _env: Env, _msg: SudoMsg) -> StdResult<Response> {
+pub fn cusom_entrypoint(deps: DepsMut, _env: Env, _msg: SudoMsg) -> StdResult<Response> {
```
Comment (jawoznia) [inline src/advanced/entry_points_overriding.md]:
```suggestion
pub fn custom_entrypoint(deps: DepsMut, _env: Env, _msg: SudoMsg) -> StdResult<Response> {
```

File: src/advanced/entry_points_overriding.md [hunk 5]
```diff
@@ -1,29 +1,30 @@
...
     CounterContract::new().counter.save(deps.storage, &3)?;
     Ok(Response::new())
 }
 ```
 
-You have to define the `SudoMsg` yourself, as it is not yet supported.
+You have to define the `CustomEntrypointMsg` yourself, as it is not yet supported.
```
Comment (jawoznia) [inline src/advanced/entry_points_overriding.md:22]: Maybe `CustomSudoMsg` instead?

File: src/advanced/sudo.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
+# Sudo entry point
+
+Sylvia supports a `sudo` type entry point both in interfaces and in
+contracts. Those methods can be used as a part of the network's
+governance procedures. More informations can be found in official
+CosmWasm documentation. From ^sylvia user point of view there's no
```
Comment (jawoznia) [inline src/advanced/sudo.md]: Provide link to the documentation.

File: src/basics/first-messages.md [hunk 7]
```diff
@@ -73,17 +73,3 @@ and then run:
 ```shell
 contract $ cosmwasm-check target/wasm32-unknown-unknown/release/contract.wasm
 ```
-
```
Comment (jawoznia) [inline src/basics/first-messages.md:76]: Paste the passing output here.

File: src/basics/good-practices.md [hunk 8]
```diff
@@ -173,17 +142,21 @@ edition = "2021"
 [lib]
 crate-type = ["cdylib", "rlib"]
 
+[features]
+library = []
+
 [dependencies]
-cosmwasm-std = { version = "1.3.1", features = ["staking"] }
-sylvia = { path = "../../confio/sylvia/sylvia" }
```
Comment (jawoznia) [inline src/basics/good-practices.md:178]: thank you :D

**Review by jawoznia — APPROVED** [review]
Some minor comments.
Merge when resolved.

File: src/advanced/attributes_forwarding.md (hunk 1 above)
Comment (kulikthebird) [reply src/advanced/attributes_forwarding.md]: I'll go for `any` instead of `every`

File: src/advanced/entry_points_overriding.md (hunk 5 above)
Comment (kulikthebird) [reply src/advanced/entry_points_overriding.md:22]: I didn't want to refer to sudo here as it is fully supported by Sylvia. It should be treated as an extra feature that helps with any new CosmWasm entrypoints that will appear in the future and won't be handled by Sylvia at that point.

File: src/basics/good-practices.md (hunk 8 above)
Comment (kulikthebird) [reply src/basics/good-practices.md:178]: you're welcome!

**Pull request merged**